gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/JobInfo.java

```diff
@@ -78,16 +88,44 @@ public JobInfo apply(Job pb) {
     WRITE_EMPTY
   }
 
-  private final String etag;
-  private final String id;
-  private final JobId jobId;
-  private final String selfLink;
-  private final JobStatus status;
-  private final JobStatistics statistics;
-  private final String userEmail;
-  private final JobConfiguration configuration;
+  /**
+   * Base class for a {@code JobInfo} builder.
```

Comment:
Nit. Though technically correct, I wonder if that is, "Base class", a good javadoc for the user which is unaware
of the internal implementation for this class builder.